go.mod

```diff
@@ -59,3 +66,5 @@ require (
 	// termdbms
 	github.com/muesli/reflow v0.3.0
 )
+
+replace github.com/hazelcast/hazelcast-go-client v1.3.1-0.20221005090358-58f17bb84ba8 => github.com/yuce/hazelcast-go-client v1.1.2-0.20221024101337-f9de25ad091f
```

Comment:
Should be removed.

Reply:
This is currently required for compact support. I will later point that to a branch in the main repo.